module/VuFind/src/VuFind/Controller/ContentController.php

```diff
@@ -74,21 +74,6 @@ public function contentAction()
             : $this->notFoundAction();
     }
 
-    /**
-     * Action called if matched action does not exist
-     *
-     * @return ViewModel
-     */
-    public function notFoundAction(): ViewModel
```

Comment:
Why is this method being removed? It is called by line 74 above!

Reply:
But the method is defined also in parent `AbstractActionController` and the behaviour is the same as far as I know.

But we can of course use a safer approach and just call `parent::notFoundAction` instead of non-existant `createConsoleNotFoundModel` method...

Reply:
I see -- I had been looking inside VuFind code but forgot that this was defined by the framework. And I believe that createConsoleNotFoundModel is a relic of the old console code, before we switched to Symfony-Console. I'm not sure why this was ever necessary, but I think you're right to remove it. I'll do a bit of testing to be sure!